AutomatticTracks/src/main/java/com/automattic/android/tracks/crashlogging/internal/SentryEventExt.kt

```diff
@@ -1,7 +1,9 @@
 package com.automattic.android.tracks.crashlogging.internal
 
+import android.annotation.SuppressLint
 import io.sentry.SentryEvent
 
+@SuppressLint("NewApi")
 internal fun SentryEvent.appendTags(tags: Map<String, String>) {
     tags.forEach { (key, value) ->
```

Comment:
Lint is complaining about using Java's `forEach` which is not available before API 24 (so we could crash on API 21).

But we're not using Java's `forEach` (which would be `tags.forEach { key, value ->`) but Kotlin's `forEach`. 

Hence, suppress.

Reply:
👋 @wzieba !

> But we're not using Java's forEach

Hmmm, indeed, this is so weird and kind of scary, why does it complain!

Minor (🔍): To be honest, at this point I would just replace it with a for loop, like this: `for ((key, value) in tags) { ... }`, just to be on the safest side, but this is totally option and minor, thus I am leaving up to you to decide if it's worth it. 🤔

Reply:
Good idea, I agree - looks safer. Fixed in 1fe16f007ee8b98cd1d326de441b82122192e4e8